### ISSUE TITLE
Fix A11y violation of Hero component from page structure organism

### ISSUE THREAD

**Issue by Elohina** [buildit/gravity-ui-web#377]
**Describe the bug**
In order to pass the accessibility tests it is required to add `alt` description to the banner image of `hero` component. This will also fix the violations of `homepage` and `page-homepage`.

**To Reproduce**
1. Go to `/gravity-ui-nunjucks`.
2. Run `npm run test:a11y`.
3. See the `hero` violation in the list of a11y violations.

**Expected behavior**
```
 Ensures <img> elements have alternate text or a role of none or presentation
 Images must have alternate text
 Help: https://dequeuniversity.com/rules/axe/3.4/image-alt?application=axe-puppeteer
```

**Comment by james-nash** [comment]
Fixed by PR #382
